extensions/azurePublish/src/deploy.ts

```diff
@@ -58,18 +58,28 @@ export class BotProjectDeploy {
       if (!language) {
         language = 'en-us';
       }
-      const publisher = new LuisAndQnaPublish({ logger: this.logger, projPath: this.projPath });
+
+      this.logger({
+        status: BotProjectDeployLoggerType.DEPLOY_INFO,
+        message: "Building the bot's resources ...",
+      });
+      await build(project, this.projPath, settings);
+
+      this.logger({
+        status: BotProjectDeployLoggerType.DEPLOY_INFO,
+        message: 'Build Success!',
+      });
 
       // this function returns an object that contains the luis APP ids mapping
       // each dialog to its matching app.
-      const { luisAppIds, qnaConfig } = await publisher.publishLuisAndQna(
+      const { luisAppIds, qnaConfig } = await publishLuisToPrediction(
```

Comment:
publishLuisToPrediction method only return luisAppIds, why we add the qnaConfig parameter here? Doesn't it affect the qna build or publish?

Reply:
Thanks @VanyLaw